app/extensions/brave/locales/en-US/preferences.properties

```diff
@@ -77,3 +77,4 @@ alwaysAllow=Always allow
 alwaysDeny=Always deny
 appearanceSettings=Appearance settings
 autoHideMenuBar=Hide the menu bar by default
+disableTitleMode=Disable Title Mode
```

Comment:
@diracdeltas I agree with you, haven't found a better name for it, so I just copied from the issue.

Should I change only i10n and keep everything else as `disableTitleMode` or change everything to `showUrlBar` instead?